src/Microsoft.AspNetCore.BuildTools.CreateLZMA/sdk/Sdk.props

```diff
@@ -0,0 +1,5 @@
+<Project>
+  <UsingTask
+    AssemblyFile="$(MSBuildThisFileDirectory)..\build\netstandard1.5\Microsoft.AspNetCore.BuildTools.CreateLZMA.dll"
```

Comment:
Need to vary the task assembly loaded based on `$(MSBuildRuntimeType)`.

Example: https://github.com/aspnet/BuildTools/blob/b56c36c46122915b86d56836e089d7e6b57838a8/src/Internal.AspNetCore.SiteExtension.Sdk/build/Internal.AspNetCore.SiteExtension.Sdk.props#L6-L12